### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by TomaszKotowski**

### CODE REVIEW

File: src/stores/LayoutStore.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { observable } from 'mobx';
+import { Bind } from 'lodash-decorators';
+
+class LayoutStore {
+
+  @observable docked = false;
```
Comment (kamil7x) [inline src/stores/LayoutStore.js]: I would use some better name, this store may be used later for other things. Like `drawerDocked` or `menuDocked`

File: src/stores/LayoutStore.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+  @Bind
+  onDock() {
```
Comment (kamil7x) [inline src/stores/LayoutStore.js]: Better name, please :)

File: src/models/navigation/navigation.view.js [hunk 3]
```diff
@@ -1,19 +1,50 @@
...
-      <NavBar
-        mode="dark"
-        rightContent={[
-          <Icon key="1" type="ellipsis" />,
-        ]}
-      >NavBar</NavBar>
+      <Drawer
+        className="my-drawer"
+        style={{ minHeight: document.documentElement.clientHeight }}
+        contentStyle={{ color: '#A6A6A6', textAlign: 'center' }}
```
Comment (kamil7x) [inline src/models/navigation/navigation.view.js]: Is this necessary?

File: src/models/navigation/navigation.view.js [hunk 4]
```diff
@@ -1,19 +1,50 @@
...
+        sidebarStyle={{ border: '1px solid #red' }}
```
Comment (kamil7x) [inline src/models/navigation/navigation.view.js]: Is this necessary?

**Review by kamil7x — CHANGES_REQUESTED** [review]

File: src/models/navigation/navigation.view.js (hunk 4 above)
Comment (TomaszKotowski) [reply src/models/navigation/navigation.view.js]: The style definition it is. The rest I will remove.

**Review by kamil7x — APPROVED** [review]

**Pull request merged**